internal/app/serve.go

```diff
@@ -0,0 +1,35 @@
+package app
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// runs goku as a web server using the port argument
```

Comment:
comments in go should start with the name of the exported variable, so `// Serve runs...`